packages/plugin-brownfield-ios/template/ios/Podfile

```diff
@@ -35,5 +31,13 @@ target 'HelloWorld' do
       :mac_catalyst_enabled => false,
       # :ccache_enabled => true
     )
+
+    # TODO: Get back to this
+    installer.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+        config.build_settings['SKIP_INSTALL'] = 'NO'
```

Comment:
What about other flags from https://www.rnef.dev/docs/brownfield/ios ?

Reply:
I tried removing this code block, and it looks like it's working well. Not sure what this exactly solved but removing it as a whole for now.

cc: @okwasniewski